src/sass/_storage_pool_selector.scss

```diff
@@ -0,0 +1,43 @@
+.storage-pool-select-dropdown {
+  max-width: 50rem;
+
+  .p-list {
+    width: auto;
+  }
+
+  .header {
+    background-color: $colors--theme--background-alt;
+    display: flex;
+    gap: $sph--small;
+    padding: $sph--x-small $sph--small;
+    position: sticky;
+    top: 0;
+
+    span {
+      @extend %table-header-label;
+
+      color: $colors--theme--text-default;
```

Comment:
```suggestion
```
I think this can be removed or why was it added?

Reply:
I don't remember adding this, it must have been an accident, apologies.